README.md

```diff
@@ -1,68 +1,51 @@
-# EVE
 
-This project aims to simulate basic lifeforms evolving and getting smarter, in a simplified manner.
+# EVE
 
-## Development
+[![experimental](http://badges.github.io/stability-badges/dist/experimental.svg)](http://github.com/badges/stability-badges)[![unstable](http://badges.github.io/stability-badges/dist/unstable.svg)](http://github.com/badges/stability-badges)
 
-### Setup
+##### Simulate simple, artificial lifeforms as they learn and evolve
 
-1. Install cmake & [Catch2](https://github.com/catchorg/Catch2) on your OS
```

Comment:
doesnt mention you have to install catch2 in setup

Reply:
I think for building and running eve you don't actually need catch2, only for setting up the testing.
That's why I put it under collaboration only

Reply:
True, but if you setup the repo on your PC, you probably want to develop. Otherwise you would simply download a Release once that's ready.

Reply:
true
then I have to reformulate it in general a bit though